audpsychometric/core/reliability.py

```diff
@@ -126,6 +126,9 @@ def intra_class_correlation(
     The model is based on analysis of variance,
     and ratings must at least be ordinally scaled.
 
+    CCC_ is conceptually and numerically related to the ICC.
```

Comment:
I am fine with the longer discussion being removed.
Unfortunately no progress in elaborating the conceptual and numerical relationships in greater depth...

Reply:
This was only a statement. Nothing to act upon. 
Resolving the conversation.